README.md

```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
```

Comment:
### Left reviews and line comments
**Created a file and made a commit**
_Opened a pull request_